File: RecordMusicCunoc/src/main/java/com/adolfosc/modelo/conexion/Dato.java
package com.adolfosc.modelo.conexion;

/**
 *
 * @author hectoradolfo
 */
public class Dato {
    
    private String canal;
    private String nota;
    private String octava;
    private String duracion;

    public Dato() {
        this.canal = "";
        this.nota = "";
        this.octava = "";
        this.duracion = "";
    }

    public String getCanal() {
        return canal;
    }

    public void setCanal(String canal) {
        this.canal = canal;
    }

    public String getNota() {
        return nota;
    }

    public void setNota(String nota) {
        this.nota = nota;
    }

    public String getOctava() {
        return octava;
    }

    public void setOctava(String octava) {
        this.octava = octava;
    }

    public String getDuracion() {
        return duracion;
    }

    public void setDuracion(String duracion) {
        this.duracion = duracion;
    }
}

File: RecordMusicApp/app/src/main/java/com/adolfosc/modelo/conexion/Dato.kt
package com.adolfosc.modelo.conexion

class Dato {

    private var canal: String? = null
    private var nota: String? = null
    private var octava: String? = null
    private var duracion: String? = null

    constructor() {
        canal = ""
        nota = ""
        octava = ""
        duracion = ""
    }

    fun Dato() {
        canal = ""
        nota = ""
        octava = ""
        duracion = ""
    }

    fun getCanal(): String? {
        return canal
    }

    fun setCanal(canal: String?) {
        this.canal = canal
    }

    fun getNota(): String? {
        return nota
    }

    fun setNota(nota: String?) {
        this.nota = nota
    }

    fun getOctava(): String? {
        return octava
    }

    fun setOctava(octava: String?) {
        this.octava = octava
    }

    fun getDuracion(): String? {
        return duracion
    }

    fun setDuracion(duracion: String?) {
        this.duracion = duracion
    }
}